node-chromium/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:lts-alpine
+
+RUN apk add --no-cache \
```

Comment:
Sorry I missed this pre-merge.

Should we && `apk update && apk upgrade` before this?

Reply:
was subsequently added https://github.com/alphagov/paas-docker-cloudfoundry-tools/pull/218/files